### PR TITLE
add Dataverse documentation

### PR DESCRIPTION
**Pull request by joel-simpson**
Adds Dataverse Integration page providing an overview of Archivematica to Dataverse integration. Update the Integrations page and Transfers page with relevant references / links

connected to https://github.com/artefactual/archivematica/issues/1086
connected to https://github.com/artefactual/archivematica/issues/1147

### CODE REVIEW

**Review by sallain — COMMENTED** [review]
I think this looks good, @joel-simpson! @sevein noticed a couple of places where there's a period missing, so I'll add some comments to reflect that.

If you have time, can you squash the commits? It would be nice to merge this as one commit instead of 9.

File: admin-manual/installation-setup/integrations/integrations.rst [hunk 1]
```diff
@@ -50,4 +51,15 @@ Arkivum
...
 
+.. _dataverse-integration:
+
+Dataverse
+---------
+
+See :ref:`Dataverse integration <dataverse-transfers>` for an overview of how 
+Dataverse datasets are preserved. 
+For instructions to configure a Dataverse integration, see 
+:ref:`Storage Service docs <storageservice:administrators>`
```
Comment (sallain) [inline admin-manual/installation-setup/integrations/integrations.rst]: Needs a period.

File: user-manual/transfer/dataverse.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+events using the eventOutcomeDetailNote attribute to indicate the source of 
+the checksum.
+
+.. _dataverse-configuration:
+
+Configuration
+-------------
+
+Integration with a Dataverse repository is configured in the Storage Service. 
+For detailed instructions, see the :ref:`Administrators Manual <storageservice:administrators>`
```
Comment (sallain) [inline user-manual/transfer/dataverse.rst]: Needs a period.

File: user-manual/transfer/transfer.rst [hunk 3]
```diff
@@ -54,6 +54,9 @@ transfer(s). Once this is done, the SIP can be moved into ingest or sent to a
 
    If your transfer is composed of digital forensic disk images, please see
    :ref:`Forensic image processing <forensic>`.
+   
+   If your transfer is a Dataverse dataset, please see 
+   :ref:`Dataverse Integration <dataverse>`
```
Comment (sallain) [inline user-manual/transfer/transfer.rst]: Needs a period

File: admin-manual/installation-setup/integrations/integrations.rst (hunk 1 above)
Comment (joel-simpson) [reply admin-manual/installation-setup/integrations/integrations.rst]: agreed

File: user-manual/transfer/dataverse.rst (hunk 2 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst]: yep

File: user-manual/transfer/dataverse.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+.. _dataverse-overview:
+
+Overview
+--------
+
+`Dataverse`_ is an open source repository for research data. Archivematica can
+be configured to use a Dataverse Repository as a Transfer Source Location. 
+Dataverse Transfer Source Locations can be configured to display all available 
+datasets or a subset using query parameters.
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst]: Suggested: `subset of them`.

**NB.** Depending on how a query is configured a user will only see the first 50 results returned.

File: user-manual/transfer/dataverse.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+* :ref:`Dataverse METS file <dataverse-mets>`
+* :ref:`AIP METS file <dataverse-aip-mets>`
+* :ref:`Configuration <dataverse-configuration>`
+
+.. _selecting-datasets:
+
+Selecting Datasets for preservation
+-----------------------------------
+
+When a Dataverse Transfer Source Location is selected in the Transfer Tab of
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst]: I'm not sure about the capitalization of 'Transfer Tab'

File: user-manual/transfer/dataverse.rst [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+Archivematica generates a Dataverse METS file that describes the contents of 
+the dataset as retrieved from Dataverse. The Dataverse METS includes:
+
+* descriptive metadata about the dataset, mapped to the `DDI standard`_
+
+* a <mets:fileSec> section that lists all files provided, grouped by type 
+  (original, metadata or derivative)
+
+* a <mets:structMap> section that describes the structure of the files as
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst]: For snippets of xml like this, double inverted quotations are a good idea to make it clearer to read, e.g. ` ``<mets:fileSec>`` `.

image

File: user-manual/transfer/dataverse.rst [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+Selecting Datasets for preservation
+-----------------------------------
+
+When a Dataverse Transfer Source Location is selected in the Transfer Tab of 
+the Dashboard, users can browse a list of available datasets. Selecting a 
+directory icon will expand the view to display the list of files included in 
+the dataset. Individual files can’t be selected for Transfer.
+
+When a Dataverse dataset is selected, the Transfer Type ‘Dataverse’ must also
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst]: Similarly, not sure about capitalization of transfer type.

File: user-manual/transfer/dataverse.rst [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+* descriptive metadata about the dataset, mapped to the `DDI standard`_
+
+* a <mets:fileSec> section that lists all files provided, grouped by type 
+  (original, metadata or derivative)
+
+* a <mets:structMap> section that describes the structure of the files as 
+  provided by Dataverse (particularly helpful for understanding which files were
+  provided in `Tabular Data file bundle`_)
+
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst:94]: I think this is useful. There seems to be a few connecting words missing. Suggest:  `this is particularly helpful for understanding the files that are provided in a tabular data file 'bundle'`. 

But I'd also introduce a bundle somewhere else, maybe in describing the feature?

File: user-manual/transfer/dataverse.rst [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+The Dataverse METS is found in the final AIP in this location: 
+<AIP Name>/data/objects/metadata/transfers/<transfer name>/METS.xml
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst]: Like above, path strings fit well inside double inverted commas ` ``path/to/something`` `.

File: user-manual/transfer/dataverse.rst [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+* a <mets:fileSec> section that lists all files provided, grouped by type 
+  (original, metadata or derivative)
+
+* a <mets:structMap> section that describes the structure of the files as 
+  provided by Dataverse (particularly helpful for understanding which files were
+  provided in `Tabular Data file bundle`_)
+
+The Dataverse METS is found in the final AIP in this location: 
+<AIP Name>/data/objects/metadata/transfers/<transfer name>/METS.xml 
+(This is also where you will find the dataset.json metadata file provided by
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst:97]: As we haven't introduced `dataset.json` we might change the definitive `the` for the indefinite `a`.

File: user-manual/transfer/dataverse.rst [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+Dataverse, and the agents.json metadata file created by Archivematica).
+
+.. _dataverse-aip-mets:
+
+AIP METS file
+-------------
+
+The Archival Information Package (AIP) METS file follows the basic structure 
+for a standard Archivematica AIP METS file. Derivatives generated by Dataverse
+are indicated using the METS fileGrp attribute (where USE =“derivative”).
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst]: Suggest ` ``USE="derivative"`` ` (double inverted commas). For the dmdSecs in brackets below as well.

File: user-manual/transfer/dataverse.rst [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+
+The descriptive metadata (dmdSecs) in the Dataverse METS file are copied over 
+to the AIP METS file.
+
+In the PREMIS Object entity, relationships between original and derivative 
+tabular format files from Dataverse are described using PREMIS relationship 
+semantic units. A PREMIS derivation event indicates the derivative file was
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst]: Possibly just need `... PREMIS semantic units.`.

File: user-manual/transfer/transfer.rst [hunk 13]
```diff
@@ -54,6 +54,9 @@ transfer(s). Once this is done, the SIP can be moved into ingest or sent to a
 
    If your transfer is composed of digital forensic disk images, please see
    :ref:`Forensic image processing <forensic>`.
+   
+   If your transfer is a Dataverse dataset, please see 
+   :ref:`Dataverse Integration <dataverse>`.
```
Comment (ross-spencer) [inline user-manual/transfer/transfer.rst:59]: Dataset seems a little diminutive, although I appreciate that is technically what we're requesting from the API. We've seen fairly significant transfers from just our demo site. 

I would suggest the text:

`If your transfer is from a Dataverse repository, please see ...`

**Review by ross-spencer — CHANGES_REQUESTED** [review]
Hi @joel-simpson. This is pretty good. The main things I have spotted are in your placement of xml snippets or OS paths inline. These display a little better using double inverted commas. There are a few other small suggestions I've made along the way.

File: user-manual/transfer/dataverse.rst (hunk 5 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst]: good spot. It looks like in the rest of the docs 'Transfer tab' is used so am changing to that.

File: user-manual/transfer/dataverse.rst (hunk 6 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst]: thanks for the suggestion. Have updated these with the additional quotations.

File: user-manual/transfer/dataverse.rst (hunk 7 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst]: On the Transfer page of our docs, neither is capatilized so have corrected to be consistent.

File: user-manual/transfer/dataverse.rst (hunk 9 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst]: thanks, now updated.

File: user-manual/transfer/dataverse.rst (hunk 10 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst:97]: agreed

File: user-manual/transfer/dataverse.rst (hunk 11 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst]: yep

File: user-manual/transfer/dataverse.rst (hunk 12 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst]: yes, that does read better. (less is more)

File: user-manual/transfer/dataverse.rst (hunk 8 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst:94]: good suggestions. I updated the wording as suggested, and also added a section on dataset contents - to highlight that we retrieve all 'bundles'.

File: user-manual/transfer/dataverse.rst (hunk 10 above)
Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst:97]: actually changed back to 'the' after adding a section describing dataset contents (where I now introduce the dataset.json file)

File: user-manual/transfer/transfer.rst (hunk 13 above)
Comment (joel-simpson) [reply user-manual/transfer/transfer.rst:59]: that's interesting... I actually think of many datasets being massive (e.g. astronomy examples I've heard of that are TB big). But your wording suggestion works just as well so have updated to that.

Comment (ross-spencer) [reply user-manual/transfer/transfer.rst:59]: Good point! One of the first presentations I saw in this field was CERN. They had a little bit of data I suppose ;)

File: user-manual/transfer/dataverse.rst (hunk 8 above)
Comment (sallain) [reply user-manual/transfer/dataverse.rst:94]: Sorry for one more nitpick - there's a hanging ) in line 93

Comment (joel-simpson) [reply user-manual/transfer/dataverse.rst:94]: well spotted. now fixed.

**Comment by sallain** [comment]
Thanks Joel! This has been merged into 1.8 [here](https://github.com/artefactual/archivematica-docs/commit/20dd22faa5438cd9557a8e948749abb751f4cdf9).

**Pull request closed without merging**